docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+# Glossary
+
+Definition of commonly used terms in the context of this Romcal project. The terms are listed in alphabetical order.
+
+## Celebration
+
+A celebration is a liturgical or sacramental event, typically consisting of rituals, readings, prayers, and often the Eucharist (Mass), whether or not it is associated with a liturgical day.
+
+Note: Currently, Romcal does not provide specific information about celebrations, such as the list of Masses for a given day or the hours of the Divine Office. Instead, it indicates unique liturgical days determined within a liturgical calendar. However, the ability to list these celebrations—linked to a liturgical day or corresponding to a civil day (such as Vigil Masses)—is a feature that may be added in the future.
+
+## General Instruction of the Liturgy of the Hours (GILH)
+
+The General Instruction of the Liturgy of the Hours (GILH) is the document that provides the general instructions for the Liturgy of the Hours. It is an official liturgical document that was promulgated by the Congregation for Divine Worship and the Discipline of the Sacraments.
+
+An English translation of the GILH can be found [here](https://divineoffice.org/general-instructions/).
+
+## General Instruction on the Roman Missal (GIRM)
```

Comment:
```suggestion
## General Instruction on the Roman Missal
```

Reply:
Here, I think it's important to keep the acronym in the title to keep it prominent. Because this acronym is what we'll use elsewhere to refer to this document.

Reply:
i'm not gonna comment on all of them, but i'm gonna resolve the ones that are all this same change.